openassessment/templates/openassessmentblock/student_training/student_training.html

```diff
@@ -30,7 +30,8 @@ <h2 class="step__title">
           <span class="step__status__label">{% trans "This step's status" %}:</span>
           <span class="step__status__value">
               <span class="copy">{% trans "In Progress" %}
-              (<span class="step__status__value--completed">{{ training_num_completed }}</span> of
+              (<span class="step__status__value--completed">{{ training_num_completed }}</span>
+                {% trans "of" %}
```

Comment:
I'm concerned that this will rarely be the correct translation.  I'm not sure what tools are available to us in Django to build these from strings like `"({training_num_completed} of {training_num_available})"`.